NOTES.md

```diff
@@ -1,3 +1,191 @@
+    * Removed the datacenter abstraction and changed the arguments to `tableCreate` (#2876)
+    * Added a `reconfigure` command to change the replication and sharding settings of a table (#2932)
+    * Added a `rebalance` command to even out the size of a table's shards (#2981)
+    * Added a `config` command for tables and databases as an alias for the corresponding row in `db_config` or `table_config`
+    * Added a `status` command for tables and databases as an alias for the corresponding row in `db_status` or `table_satus`
+    * Redundant information has been removed from the `/ajax` endpoint (#2878, #2879)
+    * Automatically migrate data directories to the new semilattice configuration format (#2869)
+    * The stats now contains the number of client connections (#2989)
+    * Added more information to the return value of `db_create`, `db_drop`, `table_create` and `table_drop` (#3001)
+    * Changed how `durability` and `write_acks` are configured (#3066)
+    * Now allow changing the cache size at runtime (#3166)
+    * Now expose the command line of the server (#3180)
+    * Improved the scalability for large table creation and reconfiguration in large clusters (#3198)
+    * Added a new UI for table configuration (#3229)
+    * Now allow over-sharding tables when there are not enough documents (#3271)
+* ReQL
+  * Added `r.range` which generates all numbers from a given range (#875)
+  * Enforce UTF-8 encoding on input strings (#1181)
+  * Added a `wait` command which waits for a table to be ready (#2259)
+  * Added `toJsonString` which converts a datum into a JSON string (#2513)
+  * Turned `map` into a variadic function for mapping over multiple sequences in parallel (#2574)
+  * Added a prefix version of `map` (#3321)
+  * Added an optional `squash` argument to the `changes` command which deafaults to `true`, letting the sever combine multiple changes to the same document (#2726, #3558)
+  * `min` and `max` will now use an index when passed the new `index` parameter (#2974, #2981)
+  * Improved changefeeds
+    * `changes` can now be placed after `between` and `filter` (#3232)
+    * `changes` can also be used on a few other commands for which there should be a comprehensive list here
+* Server
+  * Made buffered IO the default, added a `--direct-io` flag to enable direct IO and deprecated the `--no-direct-io` flag (#3391)
+* Python driver
+  * `rethinkdb export` now exports secondary index information and `rethinkdb import` re-creates those indexes unless it is given the `--no-secondary-indexes` flag (#3484)
+
+## Improvements ##
+
+* Web UI
+  * The web assets are now compiled into the `rethinkdb` executable (#1093)
+  * `getAll` queries on secondary indexes are now shown in the performance graph (#2379)
+  * Added `getField` to the autocompletion in the data explorer (#2624)
+  * Added support for changefeeds in the data explorer (#2643)
```

Comment:
I might say:

> Added live updates for changefeeds in the Data Explorer

...since support is a weak verb that doesn't communicate how _awesome_ this feature is.